Tests/js/models/assets/ez-contentmodel-tests.js

```diff
@@ -1548,6 +1548,46 @@ YUI.add('ez-contentmodel-tests', function (Y) {
         },
     });
 
+    toObjectTest = new Y.Test.Case({
+        name: "eZ Content Model toObject test",
+
+        setUp: function () {
+            this.fieldsByLanguage = {fields: 'paracétamole'};
+            this.currentVersion = new Y.Mock({Version: {VersionInfo: {},Fields: {field :[]}}});
```

Comment:
that's right, but actually `this.currentVersion` should be the object you pass in `Y.Mock` constructor parameter. You don't need a Mock and then the expectation in the following line is also not needed (this is what confused me)